exercises/phone-number/README.md

```diff
@@ -13,7 +13,7 @@ The format is usually represented as
 
 where `N` is any digit from 2 through 9 and `X` is any digit from 0 through 9.
 
-Your task is to clean up differently formated telephone numbers by removing punctuation and the country code (1) if present.
+Your task is to clean up differently formatted telephone numbers by removing punctuation and the country code (1) if present.
```

Comment:
This change was made in a separate PR that was already merged into master, no need to include it here.